tt_eager/tracy.py

```diff
@@ -29,6 +29,8 @@
 
 import tracy_state
 
+DEFAULT_CHILD_CALLS = ["CompileProgram", "HWCommandQueue_write_buffer"]
```

Comment:
@davorchap @cglagovichTT We can ask for child calls in command line but this line here is a list of functions that will always be included for the op if it is present. What are the child functions you are interested in that should be included by default?